src/StdCheats.sol

```diff
@@ -569,4 +569,22 @@ abstract contract StdCheats is StdCheatsSafe {
             stdstore.target(token).sig(0x18160ddd).checked_write(totSup);
         }
     }
+
+    function isFork() internal virtual returns (bool status) {
+        try vm.activeFork() {
+            status = true;
+        } catch (bytes memory) {}
+    }
+
+    modifier skipForking() {
```

Comment:
Nit on naming, wdyt of `skipWhenForking` and `skipWhenNotForking` instead (or `skipIfForking` and `skipIfNotForking`)? `skipForking` is a bit less clear, e.g. can be interpreted as the modifier resulting in fork-related cheats not executing / having different behavior.

Otherwise LGTM, test failure unrelated.

Reply:
ya, `when` is better naming. changed 👍